samples: protocols_serialization: use different GPIO pins of UART (#18608)
nrfconnect/sdk-nrf: samples/nrf_rpc/protocols_serialization/client/README.rst

To test the client sample, follow the instructions in the :ref:`protocols_serialization_server_sample_testing` section of the protocol serialization server sample test procedure.

.. note::

>    When using the nRF54L15 DK, you shall not press **Button 1** nor **Button 2**, since GPIO pins connected to these buttons are used by the UART peripheral for communication with the server device.

Reviewer: ```suggestion
   When using the nRF54L15 DK, do not press **Button 1** or **Button 2**.
   The GPIO pins connected to these buttons are used by the UART peripheral for communication with the server device.
```
